src/tests/unittests/qiskit/test_passes.py

```diff
@@ -204,6 +204,18 @@ def test_LinearTopologyParallelPass(self):
         self.assert_circuit_equivalence(qc_transpiled, qc_target)
         self.assertEqual(circuit_instruction_names(qc_transpiled), circuit_instruction_names(qc_target))
 
+    def test_RemoveGateByName(self):
+        qc = QuantumCircuit(3)
+        qc.h(0)
+        qc.x(1)
+
+        qc_transpiled = RemoveGateByName('none')(qc)
+        self.assertEqual(circuit_instruction_names(qc_transpiled), circuit_instruction_names(qc))
+
+        for name in ['x', 'h', 'dummy']:
```

Comment:
name is not used. But 'h' is hardcoded. Maybe change 'h' to name in line 216 and 217